packages/tempus/src/Tempus_StepperExplicitRK_impl.hpp

```diff
@@ -238,9 +246,9 @@ void StepperExplicitRK<Scalar>::initialize()
     "Error - Need to set the model, setModel(), before calling "
     "StepperExplicitRK::initialize()\n");
 
-  this->setTableau(this->stepperPL_);
-  this->setParameterList(this->stepperPL_);
-  this->setObserver();
+  //this->setTableau(stepperPL_);
+  //this->setParameterList(stepperPL_);
+  //this->setObserver();
 
```

Comment:
@sconde these commented out lines should be restored; I included them by mistake when I sent you the patch file. This could be the source of the Drekar failures you are seeing.